site/themes/embark/layout/docs.swig

```diff
@@ -51,7 +51,7 @@
       </div>
       <div class="o-guided-content__content">
         <p class="c-meta">{{ __('page.last_updated', date(page.updated, 'MMMM Do YYYY')) }}
-          (<a href="https://github.com/embark-framework/embark-site/edit/master/source/{{page.source}}" target="_blank">
+          (<a href="https://github.com/embark-framework/embark/edit/master/site/source{{page.source}}" target="_blank">
```

Comment:
🤔need to test this, but isn't there a `/` missing after `source` ?

Reply:
yeah it's my mistake it's missing a `/`, should I open a new PR or fix it and push after my last commit ?

Reply:
I fixed it I wish this time I didn't do another mistake 😓

Reply:
@alchemist107 this is perfect. Thanks man!